packages/extension/src/providers/ethereum/networks/opl.ts

```diff
@@ -0,0 +1,24 @@
+import wrapActivityHandler from "@/libs/activity-state/wrap-activity-handler";
+import { EvmNetwork, EvmNetworkOptions } from "../types/evm-network";
+import { EtherscanActivity } from "../libs/activity-handlers";
+import { NetworkNames } from "@enkryptcom/types";
+
+const oplOptions: EvmNetworkOptions = {
+  name: NetworkNames.Opal,
+  name_long: "Opal EVM",
+  homePage: "https://unique.network/",
+  blockExplorerTX: "https://uniquescan.io/opal/tx/[[txHash]]",
+  blockExplorerAddr: "https://uniquescan.io/opal/account/[[address]]",
+  chainID: "0x22b2",
+  isTestNetwork: true,
+  currencyName: "OPL",
+  currencyNameLong: "Opal",
+  node: "wss://ws-opal.unique.network",
+  icon: require("./icons/opal.svg"),
+  gradient: "#0CB6B8",
+  activityHandler: wrapActivityHandler(EtherscanActivity),
```

Comment:
Also it looks like you didnt add the proper blockexplorer urls to `https://github.com/enkryptcom/enKrypt/blob/main/packages/extension/src/providers/ethereum/libs/activity-handlers/providers/etherscan/configs.ts#L3`
if you dont have an api thats compatible with `etherscan` you have to create you own activity handler